scalastyle-config.xml

```diff
@@ -92,7 +92,7 @@ This file is divided into 3 sections:
  */
 
 \E)?\Q/*
- * Copyright (2020) The Hyperspace Project Authors.
+ * Copyright (\E\d{4}(?:-\d{4})?\Q) The Hyperspace Project Authors.
```

Comment:
This will not catch invalid years?

Reply:
I think the purpose of having a code linter is mainly to catch human errors, and having a strict validation here seems like overkill. But if you want we can make it something like 20\d{2} to accept any 20xx year (although we can't still prevent invalid years like 2099 or 2099-2010), or make it accept only years known to valid for now (2020|2021|2020-2021) and change the format every year.

Reply:
I definitely prefer catching this kind of error via scalastyle (why waste time in PR?). We can just do `(2020|2021)` and I don't think it's an overkill to update this every year.

Reply:
After some research it seems sufficient to use something like (2020|2021|...|<current year>). When you create a new file, you just write the year at the moment. No need to update old files.

Reply:
By the way, if you really want to be strict, then we should prevent people using past year when creating new files and updating the year of old files - currently we can't catch those errors

Reply:
I'm also ok with adding new years as we get into a new year. Also, I would not want to have ranges (ie: `2019-2021`). The year of first publishing should be enough.